test/plug/router_test.exs

```diff
@@ -144,6 +144,26 @@ defmodule Plug.RouterTest do
       resp(conn, 200, inspect(conn.assigns))
     end
 
+    post "/options/assigns", assigns: %{an_option: :a_value} do
```

Comment:
We don't need to test the assigns per-se. I think just checking the route returns a 200 is enough. Can you please amend? Thanks!

Reply:
Sure. I'll do that.